test/src/com/xilinx/rapidwright/edif/TestEDIFNetlist.java

```diff
@@ -327,4 +329,50 @@ public void testBussedPortNamingCollision(@TempDir Path path) {
         EDIFPort testPort1 = testTopCell.getNet(net1.getName()).getPortInst(null, portName + "[0]").getPort();
         Assertions.assertEquals(port1.getWidth(), testPort1.getWidth());
     }
+
+    @Test
+    public void testGetIOStandard() {
+        final EDIFNetlist netlist = EDIFTools.createNewNetlist("test");
+
+        EDIFCell top = netlist.getTopCell();
+        EDIFPort port = top.createPort("O", EDIFDirection.OUTPUT, 1);
+        EDIFCellInst obufds = top.createChildCellInst("obuf", Design.getPrimitivesLibrary().getCell("OBUFDS"));
+        EDIFNet net = top.createNet("O");
+        new EDIFPortInst(port, net);
+        new EDIFPortInst(obufds.getPort("O"), net, obufds);
+        Assertions.assertEquals(EDIFNetlist.DEFAULT_PROP_VALUE, netlist.getIOStandard(obufds));
+
+        netlist.cellInstIOStandardFallback = null;
```

Comment:
```suggestion
        // Previous call to netlist.getIOStandard() will have initialized this map,
        // clear it here so that it gets re-initialized
        netlist.cellInstIOStandardFallback = null;
```